devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -23,7 +23,7 @@ metadata {
 		capability "Light"
 
 		// Generic
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008"
+		fingerprint profileId: "0104", deviceId: "0101", inClusters: "0006, 0008", deviceJoinName: "Generic Dimmable Light"
```

Comment:
Same regarding ocfDeviceType: "oic.d.light"

Reply:
I didn't do it here because it's already set at the DTH level

Reply:
@tpmanley D'oh! Of course. :)